lms/djangoapps/lti_provider/views.py

```diff
@@ -107,6 +109,34 @@ def lti_launch(request, course_id, usage_id):
     # used earlier to verify the oauth signature.
     store_outcome_parameters(params, request.user, lti_consumer)
 
+    # Make a copy of params for the event signal, and remove sensitive oauth parameters.
+    launch_params = params.copy()
+    for key in list(launch_params.keys()):
+        if key.startswith('oauth_'):
+            launch_params.pop(key)
```

Comment:
Why are we removing these params since nowhere do I see we are directly passing `params`?